docker/docker_image_src.go

```diff
@@ -190,6 +190,7 @@ func (s *dockerImageSource) fetchManifest(ctx context.Context, tagOrDigest strin
 	if err != nil {
 		return nil, "", err
 	}
+	logrus.Debugf("Content-Type from GET to %s is %s", path, res.Header.Get("Content-Type"))
```

Comment:
Please use `%q` for the header value, it can be empty or contain unexpected garbage.

(I’m not sure including `path` is all that valuable, especially because it is not a full URL; the full URL is logged anyway in `makeRequestToResolvedURLOnce`. But it does not exactly _hurt_.)

Reply:
Would then just specifying "Content-Type from manifest GET" then work?  I only included the path because it'd be clear it was a GET on the manifest itself as per the path.  I'll fix just this one piece and if you want me to drop the path I'm happy to do so.

Reply:
```console
$ skopeo linux inspect --debug docker://registry.fedoraproject.org/fedora                                                                                            
…
DEBU[0001] Ping https://registry.fedoraproject.org/v2/ status 200 
DEBU[0001] GET https://registry.fedoraproject.org/v2/fedora/manifests/latest 
DEBU[0001] GET https://registry.fedoraproject.org/v2/fedora/manifests/sha256:119d2ca164232e4e46ee07d177504d8b98172a20ffad1c11b2433a5df9494511 
DEBU[0002] Downloading /v2/fedora/blobs/sha256:d81c91deec0dc09a3f05db3f538bdba9c49f1d51e6955078f274cd4ed4376c6f 
…
```
Yes, I think adding just a `Content-Type from manifest GET` without reiterating the path would have sufficient context, although it is not trivially visible from just this part of source code.

Reply:
Should be addressed.  Thanks!